packages/carbon-web-components/src/components/data-table/data-table.scss

```diff
@@ -16,9 +17,9 @@ $css--plex: true !default;
 @use '@carbon/styles/scss/components/checkbox';
```

Comment:
Anna and i discovered we need to add missing imports for components that have styles that use `field`, `layer`, etc... i'm guessing data-table will need it. should be able to just add it to `:host(cds-table)` like : https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/10567/files#diff-61c6750e722e28b6c309405d2b52d53ede9b314acd21cf95c50130825a71c92c